features/adresseerbaarobject-embedden-in-adres.feature

```diff
@@ -0,0 +1,68 @@
+# language: nl
+Functionaliteit: Adresseerbaar object embedden in een adres
+
+	Als consumer van de BAG API
+	Wil ik bij het opvragen van Adressen tegelijk het bijbehorende Adresseerbaar Object kunnen opvragen
+	Zodat ik niet een aparte aanroep hoeft te doen voor het opvragen van het bijbehorende Adresseerbaar object
+
+Abstract Scenario: Opvragen van adressen en bijbehorend adresseerbaar object via het /adressen endpoint
+    Als GET /adressen<query part>expand=adresseerbaarObject is aangeroepen
+    Dan bevat de _embedded.adresseerbaarObject property van elk Adres in de response het bijbehorend Adresseerbaar Object
+
+    Voorbeelden:
+        | query part                                                        |
+        | ?pandidentificatie=0193100000048288&                              |
+        | ?adresseerbaarobjectidentificatie=0484010002033603&               |
+        | ?zoekresultaatidentificatie=adr-89d5a4d96f09c60716c4671fdb9334b8& |
+        | ?postcode=2391PH&huisnummer=1&                                    |
+        | /0826200000016921?                                                |
+
+Abstract Scenario: Opvragen van adressen en een deel van de kenmerken van bijbehorend adresseerbaar object via het /adressen endpoint
+    Als GET /adressen<query part>expand=adresseerbaarObject.gebruiksdoelen,adresseerbaarObject.geconstateerd,adresseerbaarObject.geometrie is aangeroepen
+    Dan bevat de _embedded.adresseerbaarObject property van elk Adres in de response de gevraagde kenmerken van het bijbehorend Adresseerbaar Object
+
+    Voorbeelden:
+        | query part                                                        |
+        | ?pandidentificatie=0193100000048288&                              |
+        | ?adresseerbaarobjectidentificatie=0484010002033603&               |
+        | ?zoekresultaatidentificatie=adr-89d5a4d96f09c60716c4671fdb9334b8& |
+        | ?postcode=2391PH&huisnummer=1&                                    |
+        | /0826200000016921?                                                |
+
+Scenario: Opvragen van adressen en bijbehorend adresseerbaarobject met overtollig expand waarden

```

Comment:
Maak van 'adresseerbaarobject' --> 'adresseerbaarObject' of net als in regel 22 van deze feature 'Adresseerbaar Object'. Indien gekozen wordt voor de eerste dan moet regel 22 wellicht ook worden aangepast.